### DISCUSSION TITLE
Make it possible to remap F-keys to mouse button 4 and 5

### DISCUSSION THREAD

**Discussion by tg3eada** [runelite/runelite#16122] [Q&A] [answered]
Is there a reason we're unable to use the extra mouse buttons for anything beyond left clicking? I'm surprised this isn't possible in the official client, but even more so with Runelite, as it seems like such an obvious thing to have. I'd rather not use AHK or some other extra program.

**Comment by Enriath** [comment, marked as answer]
[We have said in the past that we're not going to support binding things to the mouse](https://github.com/runelite/plugin-hub/pull/372#issuecomment-643683260). We recommend you use some external software to bind it, such as the software that's made for gaming mice, or AHK.

**Comment by tg3eada** [comment]
That's really unfortunate. I don't understand the problem, but it is what it is.
